django/conf/locale/th/formats.py

```diff
@@ -9,13 +9,25 @@
 MONTH_DAY_FORMAT = 'j F'
 SHORT_DATE_FORMAT = 'j M Y'
 SHORT_DATETIME_FORMAT = 'j M Y, G:i'
-# FIRST_DAY_OF_WEEK =
+FIRST_DAY_OF_WEEK = 0  # Sunday
 
 # The *_INPUT_FORMATS strings use the Python strftime format syntax,
 # see http://docs.python.org/library/datetime.html#strftime-strptime-behavior
-# DATE_INPUT_FORMATS =
-# TIME_INPUT_FORMATS =
-# DATETIME_INPUT_FORMATS =
+DATE_INPUT_FORMATS = [
+    '%d/%m/%Y',  # 25/10/2006
+    '%d %b %Y',  # 25 ต.ค. 2006
+    '%d %B %Y',  # 25 ตุลาคม 2006
```

Comment:
In Thailand it’s customary to use the [Thai solar calendar](https://en.wikipedia.org/wiki/Thai_solar_calendar) system (as it’s the official legal calendar of Thailand).

Dates and months are the same as the common Gregorian Calendar, but years are in Buddhist Era instead of the Christian/Common Era. Just add 543 to the year number when displaying, and subtract 543 from the year number when parsing.